typescript/sdk/src/warp/WarpCore.ts

```diff
@@ -619,15 +621,17 @@ export class WarpCore {
       return { amount: 'Insufficient balance for gas and transfer' };
     }
 
-    // Finally, ensure there's sufficient balance for the IGP fee, which may
-    // be a different token than the transfer token
+    // Finally, if the IGP fee token differs from the transfer token,
+    // ensure there's sufficient balance for the IGP fee
     const igpQuote = feeEstimate.interchainQuote;
-    const igpTokenBalance = await igpQuote.token.getBalance(
-      this.multiProvider,
-      sender,
-    );
-    if (igpTokenBalance.amount < igpQuote.amount) {
-      return { amount: `Insufficient ${igpQuote.token.symbol} for gas` };
+    if (!token.isFungibleWith(igpQuote.token)) {
```

Comment:
A minor change which 1) avoids an unncessary balance fetch in some cases and 2) helped me make the tests happy again without a bunch of extra stubbing